input/fsh/examples/CBTN_example.fsh

```diff
@@ -362,10 +362,10 @@ InstanceOf: NcpiFamilyRelationship
 Title: "An example family relationship based on data from CBTN"
 Usage: #example
 Description: "An example family relationship based on data from CBTN."
-* subject = Reference(PT-006SP675)
-* focus = Reference(PT-006SP660)
-* code = $family-role-code#SON "natural son"
-* status = #registered
+* patient = Reference(PT-006SP675)
+* extension[relative].valueReference = Reference(PT-006SP660)
+* relationship = $family-role-code#NCHILD "natural child"
```

Comment:
Son is redundant when we have the patient's sex. Child is a cleaner representation.